Tests/PackageGraphPerformanceTests/PackageGraphPerfTests.swift

```diff
@@ -41,7 +41,8 @@ class PackageGraphPerfTests: XCTestCasePerf {
                 path: AbsolutePath(url).appending(component: Manifest.filename),
                 url: url,
                 package: .v3(package),
-                version: "1.0.0"
+                version: "1.0.0",
+                toolsVersion: .v3_1_0
```

Comment:
Would it be worth to making a default on these, to avoid all the diffs?

Reply:
I was planning to merge this part separately.

Reply:
Don't we still want to in general leave it off, and only specify it if relevant to what the test is trying to do?

Reply:
Sure. I meant I am going (cleanly) merge this part separately. Currently, diffs in tests are just to make the compiler happy.